src/containers/WelcomeScreen/index.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+
+import { toggleWelcomeScreen } from '../../actions/appActions';
+
+const WelcomeScreenBlurBackground = styled.div`
+        width: 100vw;
+        height:100vh;
+        background: rgba(0,0,0,0.6);
+        position:absolute;
+        z-index:10;
+`;
+
+const WelcomeScreenWrapper = styled.div`
+        width: 900px;
+        height: 560px;
+        background: #ffffff;
+        border-radius:10px;
+        border: 1px solid rgba(13,28,121,0.1);
+        margin: 105px auto;
+        padding: 30px 10px;
+        text-align:center;
+`;
+
+const WelcomeScreenHeader = styled.h1`
+        font-size:45px;
+        color: #050C40;
+        font-family: Graphik;
+        line-height: 60px;	
+        font-weight:normal;
+`;
+
+const WelcomeScreenTextContent = styled.p`
+        color: rgba(5,12,64,0.6);
+        font-family: Graphik;
+        font-size: 16px;
+        line-height: 31px;
+        margin:25px auto 30px;
+        width:700px;
+`;
+
+const WelcomeScreenHeaderSecondary = styled.h2`
+        color: #050C40;
+        font-family: Graphik;
+        font-size: 23px;
+        font-weight: 500;
+        line-height: 33px;
+`;
+
+const WelcomeScreenFromScratchButton = styled.button`
+        height: 45px;
+        width: 216px;
+        border-radius: 22.5px;
+        background-color: #19C6C7;
+        display:inline-block;
+        margin: 0 15px;
+        font-size:15px;
+        color:#050C40;
+        font-family:Graphik;
+        font-weight: 600;
+        line-height: 17px;
+        text-align: center;
+        border:none;
+        cursor: pointer;
+`;
+
+const WelcomeScreenSearch = ({ className }) => (
+    <input className={className} type="text" value="Template Name"></input>
+);
```

Comment:
Similarly, this could possibly be extracted out into a `actions.js` because this component will require functionality and it could be easier to reason about if there are sections.